tap_zendesk/streams.py

```diff
@@ -133,6 +133,24 @@ def raise_or_log_zenpy_apiexception(schema, stream, e):
         raise e
 
 
+class Brands(Stream):
+    name = "brands"
+    replication_method = "INCREMENTAL"
+    replication_key = "updated_at"
+
+    def sync(self, state):
+        bookmark = self.get_bookmark(state)
+
+        brands = self.client.brands()
+        for brand in brands:
+            if utils.strptime_with_tz(brand.updated_at) >= bookmark:
+                # NB: We don't trust that the records come back ordered by
+                # updated_at (we've observed out-of-order records),
+                # so we can't save state until we've seen all records
```

Comment:
Is this something you observed, or are you copy-pasting this logic from elsewhere?